src/lsp/cobol_ptree/terms.ml

```diff
@@ -269,10 +269,42 @@ and class_ =
 
 
 and inline_call =                               (* in ancient terms: funident *)
-  {
-    call_fun: name with_loc;
-    call_args: effective_arg list;
-  }
+  | CallFunc of { (*name to avoid clash with call statement *)
+      call_fun: name with_loc;
+      call_args: effective_arg list;
+    }
+  | CallTrim of call_trim_args
+  | CallLength of call_length_args
+  | CallNumvalC of ident_or_nonnum list
+  | CallLocaleDate of locale_datetime_args
+  | CallLocaleTime of locale_datetime_args
+  | CallLocaleTimeFromSeconds of locale_datetime_args
+  | CallFormattedDatetime of formatted_datetime_args
+  | CallFormattedTime of formatted_datetime_args
+
+and call_trim_args = {
+  trimmed: effective_arg;
+  position: leading_trailing option;
```

Comment:
The name of the `leading_trailing` type is confusing. `tip: trimming_tip` could me more appropriate.